internal/responsibilities.md

```diff
@@ -13,8 +13,8 @@
 - Leon Hofmeister
 
 ## DevOps
-- Ilijaz
 - Timo
+- Ilijaz
```

Comment:
Must also be adapted in our `agile-charter`

Reply:
I added it